README.rst

```diff
@@ -2,7 +2,7 @@
 Astropy
 =======
 
-|Travis Status| |CircleCI Status| |Coverage Status| |PyPI Status| |Documentation Status|
+|Actions Status| |CircleCI Status| |Coverage Status| |PyPI Status| |Documentation Status|
```

Comment:
travis isn't completely removed yet, so maybe keep both?

Reply:
It doesn't run no more. We can add it back if we decide to pay.

Reply:
ok, indeed. I only said it based on that the config remained.